tests/unit/test_entity.py

```diff
@@ -68,6 +68,18 @@ def test_Entity_splitter():
     assert wrapper_entity.end_page == 2
 
 
+def test_Entity_classifier():
+    documentai_entity = documentai.Document.Entity(
+        type_="clinical_notes",
+        id="0",
+        confidence=0.99878639,
+    )
+    wrapper_entity = entity.Entity(documentai_entity)
+    assert wrapper_entity.type_ == "clinical_notes"
+    assert wrapper_entity.documentai_object.id == "0"
+    assert round(wrapper_entity.documentai_object.confidence, 8) == 0.99878639
+
```

Comment:
Maybe test some of the fields as well, for example if a field must remain `None` if coming from Classifier.